src/scenic/simulators/carla/model.scenic

```diff
@@ -192,32 +206,45 @@ class NPCCar(Car):  # no distinction between these in CARLA
     pass
 
 class Bicycle(Vehicle):
-    width: 1
-    length: 2
-    blueprint: Uniform(*blueprints.bicycleModels)
-
+    blueprint: Uniform(*bp.any_in("bicycle"))
+    defaultWidth: 1 
+    defaultLength: 2 
+    defaultHeight: 1.5
 
 class Motorcycle(Vehicle):
-    width: 1
-    length:2
-    blueprint: Uniform(*blueprints.motorcycleModels)
-
+    blueprint: Uniform(*bp.any_in("motorcycle"))
+    defaultWidth: 1
+    defaultLength: 2 
+    defaultHeight: 1.5
 
 class Truck(Vehicle):
-    width: 3
-    length: 7
-    blueprint: Uniform(*blueprints.truckModels)
-
-
-class Pedestrian(Pedestrian, CarlaActor, Walks, _CarlaPedestrian):
+    blueprint: Uniform(*bp.any_in("truck"))
+    defaultWidth: 2.5
+    defaultLength: 7.5
+    defaultHeight: 3
+
+class Van(Vehicle):
+    blueprint: Uniform(*bp.any_in("van"))
+    defaultWidth: 2 
+    defaultLength: 5 
+    defaultHeight: 2
+
+class Bus(Vehicle):
+    blueprint: Uniform(*bp.any_in("bus"))
+    defaultWidth: 4 
+    defaultLength: 10
+    defaultHeight: 4
+
+class Pedestrian(CarlaActor, Pedestrian, Walks, _CarlaPedestrian):
     """A pedestrian.
 
     The default ``blueprint`` (see `CarlaActor`) is a uniform distribution over the
     blueprints listed in :obj:`scenic.simulators.carla.blueprints.walkerModels`.
```

Comment:
Need to fix the docstring here, since the location of the blueprints has changed.